src/watertap_contrib/reflo/solar_models/surrogate/flat_plate/data/training_flat_plate_surrogate.py

```diff
@@ -0,0 +1,333 @@
+import os
```

Comment:
I'd like to build this functionality into the base class (i.e. running PySAM to generate data) but I want to merge this file in the meantime as a guide for how to do it for the flat plate surrogate since it is slightly different.